modules/final_radiological_review/php/NDB_Form_final_radiological_review.class.inc

```diff
@@ -180,64 +240,77 @@ class NDB_Form_final_radiological_review extends NDB_Form
                 f.CommentID as CommentID 
-                FROM radiology_review r LEFT JOIN examiners e on (e.ExaminerID=r.Examiner) 
+                FROM radiology_review r LEFT JOIN examiners e 
+                on (e.ExaminerID=r.Examiner) 
                 LEFT JOIN flag f ON (f.CommentID=r.CommentID) 
                 LEFT JOIN session s ON (s.ID=f.SessionID) 
                 LEFT JOIN candidate c ON (c.CandID=s.CandID) 
                 LEFT JOIN tarchive t ON (upper(t.PatientName) 
-                LIKE CONCAT(upper(c.PSCID), '_', upper(s.CandID), '_', upper(s.visit_label), '%')) 
+                LIKE CONCAT(upper(c.PSCID), '_', upper(s.CandID), '_',
+                upper(s.visit_label), '%')) 
                 WHERE r.CommentID=:CID 
                 GROUP by CandID, PSCID, Visit_label, SessionID, CommentID,
-                r.Scan_done, r.review_results, r.abnormal_atypical_exclusionary, r.Incidental_findings, e.full_name", 
+                r.Scan_done, r.review_results,
+                r.abnormal_atypical_exclusionary, 
+                r.Incidental_findings, e.full_name",
                 array('CID' => $this->identifier)
             );
-            if(empty($original_review)) {
+            if (empty($original_review)) {
                 $original_review = array(
-                    'Original_Scan_Done' => '',
-                    'Original_Review_Results' => '',
-                    'Original_Exclusionary' => '',
-                    'Original_Incidental_Findings' => '',
-                    'Original_Examiner' => ''
-                );
+                                    'Original_Scan_Done'           => '',
+                                    'Original_Review_Results'      => '',
+                                    'Original_Exclusionary'        => '',
+                                    'Original_Incidental_Findings' => '',
+                                    'Original_Examiner'            => '',
+                                   );
             }
 
-
             $history = $DB->pselect(
-                'SELECT userID, changeDate, col, old, new FROM final_radiological_review_history 
+                'SELECT userID, changeDate, col, old, new 
+                 FROM final_radiological_review_history
                 WHERE CommentID= :primKey ORDER BY changeDate DESC',
                 array('primKey' => $this->identifier)
             );
 
             $history_display = '';
             foreach ($history as &$history_item) {
-                $history_display .= '<tr><td>' . 
-                    $history_item['changeDate'] . '</td><td>' . 
-                    $history_item['userID'] . '</td><td>' . 
-                    $history_item['col'] . '</td><td>' . 
-                    _makePretty($history_item['col'], $history_item['old']) . '</td><td>'
-                    . _makePretty($history_item['col'], $history_item['new'])
-                    . '</td></tr>';
+                $history_display .= '<tr><td>' .
```

Comment:
Can you create a ticket that this should be moved to a template or a react component?